grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -1619,12 +1621,16 @@ class QueryService implements Status {
 
     PaginatedResponse findAllCollectionLinks(PaginationParams pagination) {
         // get total rows
-        String totalRowsQuery = 'select count(*) from consent_collection_link where deleted = 0 ' + getCollectionLinksWhereClause(pagination)
+        String totalRowsQuery = 'select count(ccl.id) from consent_collection_link ccl ' +
+                ' left join issue project on project.project_key = ccl.project_key left join issue c on c.project_key = ccl.consent_key ' +
+                ' where ccl.deleted = 0 and project.deleted = 0 and c.deleted = 0 ' + getCollectionLinksWhereClause(pagination)
```

Comment:
Generic feedback on the structure of these queries - it's very hard to read them this way. It would be a lot easier if they were broken down into a more friendly format, something like
```
'select count(ccl.id) from consent_collection_link ccl ' +
' left join issue project on project.project_key = ccl.project_key ' + 
' left join issue c on c.project_key = ccl.consent_key ' +
' where ccl.deleted = 0 ' + 
' and project.deleted = 0 ' + 
' and c.deleted = 0 ' + getCollectionLinksWhereClause(pagination)
```

Also, naming is important in these so a dev can see clearly what the join is referring to, i.e., something like:
```
'select count(ccl.id) from consent_collection_link ccl ' +
' left join issue project on project.project_key = ccl.project_key ' + 
' left join issue consent_group on consent_group.project_key = ccl.consent_key ' +
' where ccl.deleted = 0 ' + 
' and project.deleted = 0 ' + 
' and consent_group.deleted = 0 ' + getCollectionLinksWhereClause(pagination)
```